juju/python-libjuju
# [JUJU-2018] Update 2.9.36 facades & clients

#### Description

This updates the `old_client`s with the juju `2.9.36` schema.


#### QA Steps

Bootstrap with juju --channel `2.9/edge`

```bash
 $ python -m asyncio
 > from juju import model  <----------- this line ensures that all the _clients (old and new are loaded successfully)
 > m = model.Model()
 > m.connect()
 > m.deploy('ubuntu') <---------------------- these 3 lines ensure that majority of the facades are working correctly
```
